cve_bin_tool/checkers/ngircd.py

```diff
@@ -0,0 +1,24 @@
+# Copyright (C) 2023 Orange
+# SPDX-License-Identifier: GPL-3.0-or-later
+
+
+"""
+CVE checker for ngircd
+
+https://www.cvedetails.com/product/4749/Ngircd-Ngircd.html?vendor_id=2709
+https://www.cvedetails.com/product/26242/Barton-Ngircd.html?vendor_id=12890
+
```

Comment:
```suggestion

Note: Unfortunately, we can't catch some ngircd version which are on two digits (e.g. 25) 
because cve-bin-tool only extracts strings which have more than 3 characters
```

Reply:
For the time-being, I added the comment in the checker as requested